modules/prebidServerBidAdapter.js

```diff
@@ -0,0 +1,342 @@
+import Adapter from 'src/adapter';
+import bidfactory from 'src/bidfactory';
+import * as utils from 'src/utils';
+import { ajax } from 'src/ajax';
+import { STATUS, S2S } from 'src/constants';
+import { cookieSet } from 'src/cookie.js';
+import adaptermanager from 'src/adaptermanager';
+import { config } from 'src/config';
+import { VIDEO } from 'src/mediaTypes';
+import CONSTANTS from 'src/constants.json';
+
+const getConfig = config.getConfig;
+
+const TYPE = S2S.SRC;
+const cookieSetUrl = 'https://acdn.adnxs.com/cookieset/cs.js';
```

Comment:
this should be bidder agnostic, so shouldn't hard code it.

Reply:
yeah probably want to add it to the S2S options.

Reply:
adding a property called 'cookieSetUrl'